tests/Cargo.toml

```
@@ -14,5 +15,5 @@ use-compiled-tools = ["rustc_codegen_spirv/use-compiled-tools"]
 
 [dependencies]
 compiletest = { version = "0.7.0", package = "compiletest_rs" }
-rustc_codegen_spirv = { path = "../crates/rustc_codegen_spirv", version = "=0.4.0-alpha.17", default-features = false }
+rustc_codegen_spirv = { workspace = true }
```

(see the comment on `Cargo.toml` for why this suggestion)
```suggestion
rustc_codegen_spirv = { workspace = true, default-features = false }
```